rtl_433_mqtt_autodiscovery/README.md

```diff
@@ -114,7 +114,7 @@ Follow these steps to run just the autodiscovery script in a dedicated container
 docker run -e MQTT_HOST=mqtt.example.com -e MQTT_USERNAME=username -e MQTT_PASSWORD=password ghcr.io/pbkhrv/rtl_433-hass-addons-rtl_433_mqtt_autodiscovery-amd64
 ```
 
-Replace `amd64` with your appropriate architecture.
+Replace `amd64` with your appropriate architecture. For Raspberry Pi, this is `armhf`.
```

Comment:
So I think `armhf` will work on all Pis, but at least the newer ones support a newer instruction set. I have a Pi 4 running 32-bit HAOS and it's pulling `armv7l` images. I think 64-bit OS installs will pull `aarch64`.

`arch` gives me `armv7` (no `l`) so we can't just point users to that.

How about:

```suggestion
Replace `amd64` with your appropriate architecture. For Raspberry Pi, this is `armhf`, `armv7`, or `aarch64` depending on your Pi version and operating system. If unsure, running `arch` at the command line can help identify the architecture.
```